spark_parser/spark.py

```diff
@@ -494,11 +500,16 @@ def makeSet(self, token, sets, i):
 
             if parent == i:
                 continue
-
+            if self.debug['reduce']: #STRANIKS_SCAN 01/03/2016
```

Comment:
Thanks in advance for cleaning up the code. 

(By the way, git will work fine offline. And it still records who changed things and at what time.)